sdf/1.7/material.sdf

```diff
@@ -45,6 +45,10 @@
     <description>The specular color of a material specified by set of four numbers representing red/green/blue/alpha, each in the range of [0,1].</description>
   </element>
 
+  <element name="shininess" type="double" default="0" required="0">
+    <description>The specular exponent of a material</description>
+  </element>
```

Comment:
these changes need to be manually propagated to `sdf/1.8` and `sdf/1.9`

Reply:
Done in [04377ad](https://github.com/gazebosim/sdformat/pull/1016/commits/04377ad601e3def0aab4aa9b51fe2ea7cca4c40b)

It would be great to have a linter that checks for this, or even a script that automates it 🤔